src/main/java/com/gregtechceu/gtceu/common/data/materials/ElementMaterials.java

```diff
@@ -928,7 +928,7 @@ public static void register() {
                 .element(GTElements.Nt)
                 .toolStats(ToolProperty.Builder.of(180.0F, 100.0F, 65535, 6)
                         .attackSpeed(0.5F).enchantability(33).magnetic().unbreakable().build())
-                .rotorStats(24.0f, 12.0f, 655360)
+                .rotorStats(24.0f, 250.0f,12.0f, 655360)
```

Comment:
Would be great if you could change all of the other numbers to this format as well (`N.0f`)